scripts/wallet_web_start.sh

```diff
@@ -8,5 +8,11 @@ set -e
 
 echo "Running $0"
 
+# Use corresponding image based on architecture type (AMD/ARM)
```

Comment:
Should be called `amd64` or `x86` instead of `AMD` technically

Reply:
or `x86-64`

Reply:
(I would recommend `x86` or `x86-64` as they are the most commonly understood names)

Reply:
See https://en.wikipedia.org/wiki/X86-64 if you're curious where the name `amd64` came from... it's because AMD originally created the 64-bit extension to the x86 instruction set

Reply:
Looks like MySQL uses the term `amd64` on their [docker page](https://hub.docker.com/_/mysql), so perhaps that's the best one to use here to ensure clarity.